packages/tas_keybind/init.lua

```diff
@@ -0,0 +1,72 @@
+tas = tas or {}
+local oldPlayerControls = {}
+---@type table<tas.CONTROLS, Set<tas.ControlListenerCallback>>
+local controlListeners = {}
+
+---@alias tas.ControlListenerCallback fun(player: unknown, pressed: boolean):void
+---@alias Set<T> table<T, boolean>
+
+---@enum tas.CONTROLS
+tas.CONTROLS = {
+	up = 0,
+	down = 1,
+	left = 2,
+	right = 3,
+	jump = 4,
+	aux1 = 5,
+	sneak = 6,
+	dig = 7,
+	place = 8,
+	zoom = 9,
```

Comment:
nit: For enum values, we generally either use `PasalCase` or `SHOUTY_SNAKE_CASE`

Reply:
Ok, I was going with name parity for minetest though so thats why it came out like that, don't think it should matter though